src/main/java/frc/robot/Constants.java

```diff
@@ -23,16 +23,19 @@
 	public static final int RIGHT_JOYSTICK_PORT = 1;
 	public static final int MANIPULATOR_PORT = 2;
 
-	public static final int LEFT_DRIVE_MOTOR_PORT = 0;
-	public static final int RIGHT_DRIVE_MOTOR_PORT = 1;
+	public static final int LEFT_DRIVE_MOTOR_0_DEVICE_NUMBER = 10;
+	public static final int LEFT_DRIVE_MOTOR_1_DEVICE_NUMBER = 11;
+	public static final int LEFT_DRIVE_MOTOR_2_DEVICE_NUMBER = 12;
+
+	public static final int RIGHT_DRIVE_MOTOR_0_DEVICE_NUMBER = 20;
+	public static final int RIGHT_DRIVE_MOTOR_1_DEVICE_NUMBER = 21;
+	public static final int RIGHT_DRIVE_MOTOR_2_DEVICE_NUMBER = 22;
```

Comment:
What do we think of changing `DEVICE_NUMBER` to `CAN_ID` or `DEVICE_ID`?

Reply:
I like `CAN_ID`